src/data/showcases.js

```diff
@@ -1202,6 +1202,15 @@ export const Showcases = [
     source: null,
     tags: ["governance", "analytics"],
   },
+  {
+    title: "Danogo",
+    description:
+      "Danogo is a yield aggregator that provides the best place to borrow and supply on Cardano.",
```

Comment:
```suggestion
      "Danogo is a yield aggregator that provides XXX, YYY, and ZZZ on Cardano.",
```
It can generally neither be true nor verifiable that something is the "best", and you aren't offering your users a "place"... but it looks like you have some marketable products and I think they should simply be enumerated here: along with any unique features they may have.

Reply:
Thank you for your suggestion, as a yield aggregator, we get data from all lending & borrowing protocol like Liqwid, Lenfi, Danogo Bond, etc. and provide user the lowest yield for borrowing as well as the highest yield for lending, that's why we call the best. Can we change to "Danogo Yield Aggregator, easy way to Supply & Borrow on Cardano"?

Reply:
1. We can't guarantee to users that _even if_ you truly had the lowest lending rates at the time this addition was merged that this statement would be true a month, a day, or even an hour later.  It is inevitable due to copycatting, arbitrage, and timing discrepancies that nobody can guarantee the lowest rate forever.
3. So terms like "lowest", "easiest", and "best" have to be avoided: especially since promotional language doesn't have any place in these portfolio listings.  But I believe it is fair enough to say "optimized" since you have a process to optimise them.
4. Likewise we have to leave it up to users whether something is "easy"... we don't try to advise them about that here, and neither should you.  We can all assume that your web site & app will try to not to make things difficult for the users, and so any promotional language about how "easy" it is belongs on the web site & not here.

So since you've identified 2 offerings so far in additional to yield aggregation ("lending and borrowing"), I've filled in the blanks on the above suggested edit... and added a phrase indicating how you attempt to get those competitive rates ("obtaining optimized rates by combining data from multiple Cardano protocols").

Reply:
I'm committing the suggested tagline so reviewers can focus on validating the service & its user experience.